### ISSUE TITLE
Keptn's Bridge: Deeplink to project causes error (Regression after Empty state #1373)

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#1377]
After #1373 has been implemented and merged into the EAP version of Keptn's Bridge, it is no longer possible to access the project via a deeplink, e.g., https://localhost:9000/project/sockshop

The following error is printed in console:

```console
ERROR TypeError: "projects is null"
project project-board.component.ts:41
    RxJS 10
    Angular 3
    View_ProjectBoardComponent_0 ProjectBoardComponent.html:19
    Angular 23
    RxJS 5
    Angular 8
core.js:6014:19
```
